definitions/tools/hla_consensus.cwl

```diff
@@ -0,0 +1,250 @@
+#! /usr/bin/env cwl-runner
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Script to create consensus from optitype and clinical HLA typing"
+requirements:
+    - class: InlineJavascriptRequirement
+    - class: DockerRequirement
+      dockerPull: "python:3.7.4-slim-buster"
+    - class: ResourceRequirement
+      ramMin: 4000
+    - class: InitialWorkDirRequirement
+      listing:
+      - entryname: 'hla_consensus.py'
+        entry: |
+                        #This script produces 2-4 files depending on inputs and their contents
+                        #All are packaged together into a folder called hla_calls for convenience
+                        #optitype_calls.txt is always produced, and is essentially a copy of optitype's output
+                        #consensus_calls.txt is also always produced; if no clinical calls are provided, this
+                        #file is identical to optitype_calls.txt. If clinical calls are provided, they are
+                        #reproduced in clinical_calls.txt. If the clinical calls exactly match the optitype calls*, 
+                        #all 3 files described so far will contain the same information, but are not guaranteed to
+                        #be exactly the same (text ordering may differ, depending on the order calls are given in the input). 
+                        #If the clinical calls and optitype calls do not match, mismatched_calls.txt is then produced;
+                        #each line represents a gene. See below (section 'write out call files') for more mismatch details.
+                        #NOTE: optitype only produces MHC class I calls, 
+
+                        #optitype input format:
+                        #HLA-[A,B,C, ...]*xx:xx
```

Comment:
It's unclear from this comment if this is one allele per line, or if they should be comma sep, etc